src/powerbox/tools.py

```diff
@@ -447,7 +706,7 @@ def ignore_zero_ki(freq: list, kmag: np.ndarray = None):
     res_ndim = len(kmag.shape)
 
     coords = np.array(np.meshgrid(*freq[:res_ndim], sparse=False))
-    k_weights = np.any(coords != 0, axis=0)
+    k_weights = np.all(coords != 0, axis=0)
```

Comment:
I'm not sure that this is correct... I think we want to set a weight to zero if ANY of its component k's are zero.